diff --git a/node_modules/@geist-ui/react/dist/input/password.d.ts b/node_modules/@geist-ui/react/dist/input/password.d.ts
index daf98db..0e4eefe 100644
--- a/node_modules/@geist-ui/react/dist/input/password.d.ts
+++ b/node_modules/@geist-ui/react/dist/input/password.d.ts
@@ -2,6 +2,7 @@ import React from 'react';
 import { Props } from "./input-props";
 interface PasswordProps extends Props {
     hideToggle?: boolean;
+    scale?: number;
 }
 declare type NativeAttrs = Omit<React.InputHTMLAttributes<any>, keyof PasswordProps>;
 export declare type InputPasswordProps = PasswordProps & NativeAttrs;
diff --git a/node_modules/@geist-ui/react/esm/input/password.d.ts b/node_modules/@geist-ui/react/esm/input/password.d.ts
index daf98db..0e4eefe 100644
--- a/node_modules/@geist-ui/react/esm/input/password.d.ts
+++ b/node_modules/@geist-ui/react/esm/input/password.d.ts
@@ -2,6 +2,7 @@ import React from 'react';
 import { Props } from "./input-props";
 interface PasswordProps extends Props {
     hideToggle?: boolean;
+    scale?: number;
 }
 declare type NativeAttrs = Omit<React.InputHTMLAttributes<any>, keyof PasswordProps>;
 export declare type InputPasswordProps = PasswordProps & NativeAttrs;
